tests/st/utils/utils.py

```diff
@@ -147,6 +147,22 @@ def assert_no_error(self, text=None):
         if text:
             self.assert_output_contains(text)
 
+    def check_reports(self, reports):
+        """
+        Checking for unexpected reports.
+        """
+        unexpected_report = False
+        for report in reports:
+            if report in self.output:
+                unexpected_report = True
+                file_output = open("/code/calico-upgrade-report/" + report, 'rU')
+                file_output_lines = file_output.read()
+                file_output.close()
+                logger.debug("ERROR: Command generated a unexpected report:\n/code/calico-upgrade-report/%s" % report)
+                print file_output_lines
```

Comment:
why are you using print here rather than just including the output as part of the log?  I don't think we should be mixing prints with logs should we?